Sources/App/CRP/JiraService+CRP.swift

```diff
@@ -187,12 +207,15 @@ extension JiraService {
             self.environments = environments
             self.releaseType = releaseType
             self.targetDate = targetDate
-            self.businessImpact = FieldType.TextArea.Document(text: "TBD")
+            self.changeScope = FieldType.TextArea.Document(text: "TBD")
```

Comment:
Could we populate this with what I end up scraping from ticket to ticket?

```suggestion
            self.changeScope = FieldType.TextArea.Document(text: "There are a number of tickets from the Changelog that are yet to be moved to a completed status or resolution in their respective workflow. Each of these have been reviewed and commented on with why they do not impact the release, yet are in the codebase.

These tickets are:")
```